src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,32 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      {post.title
+        && (
+        <h3 className="PostInfo__title">
+          {post.title}
+        </h3>
+        )}
+      <p>
+        {' Posted by  '}
+        <UserInfo user={post.user} />
+      </p>
+    </div>
+    <p className="PostInfo__body">
+      {post.body}
+    </p>
+    {post.comments.length > 0
```

Comment:
try to convert to boolean instead of `> 0`:
```suggestion
    {!!post.comments.length
```